test/pylib/cpp/unit/unit_facade.py

```diff
@@ -39,23 +39,16 @@ def run_test(
         if not test_passed:
             allure.attach(stdout_file_path.read_bytes(), name='output', attachment_type=allure.attachment_type.TEXT)
             msg = (
-                'working_dir: {working_dir}\n'
-                'Internal Error: calling {executable} '
-                'for test {test_id} failed (returncode={returncode}):\n'
-                'output:{stdout}\n'
-                'command to repeat:{command}'
+                f'working_dir: {os.getcwd()}\n'
+                f'Internal Error: calling {executable} '
+                f'for test {test_name} failed with error ({return_code if return_code is not None else "timeout reached"}):\n'
+                f'output:{stdout_file_path.absolute()}\n'
+                f'command to repeat:{" ".join(args)}'
             )
             failure = CppTestFailure(
                 file_name.name,
                 line_num=0,
-                contents=msg.format(
-                    working_dir=os.getcwd(),
-                    executable=executable,
-                    test_id=test_name,
-                    stdout=stdout_file_path.absolute(),
-                    command=' '.join(args),
-                    returncode=return_code,
-                ),
+                contents=msg
```

Comment:
There are two changes in this hunk
1. switch from msg.format to f-strings
2. add indication of a timeout
(maybe more, btw)
Please split